README.md

```diff
@@ -40,11 +40,18 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 - [ ] What is the purpose of using _sessions_?
 
+A session is a way to store information (in variables) to be used across multiple pages.
+
 - [ ] What does bcrypt do to help us store passwords in a secure manner.
 
+Bcrypt hashes (obfuscates) passwords with salts (random data added to the hashed password).
+
```

Comment:
with JWT's we encode the tokens.  What's the difference between encoding tokens and hashing passwords?